packages/app/src/components/DocumentListItem.module.css

```
@@ -0,0 +1,108 @@
+  min-width: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-size: 13px;
+}
+
+.titleInput {
+  flex: 1;
+  min-width: 0;
+  font-size: 13px;
+  font: inherit;
```

Biome (2.4.6)

[error] Unexpected shorthand property font after font-size

(lint/suspicious/noShorthandPropertyOverrides)